packages/terra-image/src/Image.scss

```diff
@@ -1,38 +1,41 @@
-@import '~terra-mixins';
 @import './variables';
 
-.terra-Image {
-  display: block;
-  vertical-align: middle;
-}
-
-.terra-Image--default {
-  display: block;
-  vertical-align: middle;
-}
+/* stylelint-disable selector-class-pattern  */
+:local {
+  .image {
+    display: block;
+    vertical-align: middle;
+  }
 
-.terra-Image--rounded {
-  border-radius: $terra-image-rounded-border-radius;
-}
+  .default {
+    display: block;
+    vertical-align: middle;
+  }
 
-.terra-Image--circle {
-  border-radius: $terra-image-circle-border-radius;
-}
+  .rounded {
+    border-radius: var(--terra-Image-rounded-border-radius);
+  }
 
-.terra-Image--thumbnail {
-  @include terra-padding-horizontal($terra-image-thumbnail-padding-left, $terra-image-thumbnail-padding-right);
+  .circle {
+    border-radius: var(--terra-Image-circle-border-radius);
+  }
 
-  background-color: $terra-image-thumbnail-background-color;
-  border-color: $terra-image-thumbnail-border-color;
-  border-radius: $terra-image-thumbnail-border-radius;
-  border-style: $terra-image-thumbnail-border-style;
-  border-width: $terra-image-thumbnail-border-width;
-  padding-bottom: $terra-image-thumbnail-padding-bottom;
-  padding-top: $terra-image-thumbnail-padding-top;
-}
+  .thumbnail {
+    background-color: var(--terra-Image-thumbnail-background-color);
```

Comment:
Let's normalize the format for CSS custom property names to all lower-case and single hyphen to separate words. I think this will be easier to remember. I'll send out a PR to normalize the css custom props we have so far.